README.md

```diff
@@ -50,6 +50,7 @@ let pkg = rpm::PackageBuilder::new("test", "1.0.0", "MIT", "x86_64", "some aweso
         // you can set a custom mode and custom user too
         rpm::FileOptions::new("/etc/awesome/second.toml")
             .mode(rpm::FileMode::regular(0o644))
+            .caps("cap_sys_admin,cap_net_admin=pe")
```

Comment:
Is there any validation that can or should be done here?  (Genuinely a question, it might not be worth it)

Reply:
Looked into the options, RPM verifies by calling libcaps `cap_from_text()`. 
There are several crates providing this mechanism I've gone with `capctl` as it provides a FileCaps struct that will verify the string provided by a user. 

I then convert back to String for the writing of the header, hope that's an acceptable solution?

Reply:
I've looked into the validation that [RPM does](https://github.com/rpm-software-management/rpm/blob/44c76481e75342b6f9e1994d1bb7f83575341c55/lib/rpmfi.c#L789). 
I don't think that's appropriate to add here as it's only verifying that some files have been provided

Reply:
@dsteeley FYI, I think I will make this into a feature (probably a default one but I'm not certain yet) to restore the ability to compile on Windows